modules/lib/src/main/resources/assets/js/app/inputtype/ui/selector/image/ImageEditor.ts

```diff
@@ -752,6 +754,32 @@ export class ImageEditor
             }
         });
 
+        const editContainerButtonsResponsiveHandler = () => {
+            const item = new ResponsiveItem(this);
+
+            if(item.isInRangeOrSmaller(ResponsiveRanges._360_540)) {
```

Comment:
Since `editContainerButtonsResponsiveHandler` can be called numerous times on resolution changes, it will make sense to store current mode in a variable and compare it with a new mode to check if the mode has changed and any changes are required. This way we avoid setting labels/titles on controls when they should be kept the same.

Something like this:

```Typescript
this.responsiveItem = new ResponsiveItem(this);
let wasMobileMode: boolean;

const editContainerButtonsResponsiveHandler = () => {
  const isMobileMode: boolean = this.responsiveItem.isInRangeOrSmaller(ResponsiveRanges._360_540);
  if (wasMobileMode === isMobileMode) {
    return;
  }
  wasMobileMode = isMobileMode;
  if (isMobileMode) {
    // do stuff for mobile mode
  } else {
   // do stuff for desktop mode
  }
};

ResponsiveManager.onAvailableSizeChanged(this, editContainerButtonsResponsiveHandler);

```